assets/www-template/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr" class="hydrated">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Example Capacitor App</title>
+    <meta
+      name="viewport"
+      content="viewport-fit=cover, width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <meta name="format-detection" content="telephone=no" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/water.css@2/out/water.css"
+    />
+  </head>
+  <body>
+    <main>
+      <h1>Capacitor Test Plugin Project</h1>
+      <p>
+        This project can be used to test out the functionality of your plugin.
+        Nothing in the <em>example/</em> folder will be published to npm when
+        using this template, so you can create away!
+      </p>
+      <button onclick="testEcho()">Click Me!</button>
+    </main>
+
+    <script src="https://cdn.jsdelivr.net/npm/@capacitor/core@3.0.0/dist/capacitor.js"></script>
```

Comment:
Since the capacitor.config.json has `bundledWebRuntime` set to true, we can run `npx cap copy` and it will generate the capacitor.js file instead of relying on the cdn.

```suggestion
    <script src="capacitor.js"></script>
```